tests/spread/smoketests/basic/task.yaml

```diff
@@ -17,7 +18,7 @@ include:
 
 prepare: |
   charmcraft init --project-dir=charm
-  cp charmcraft-$CHARM_TYPE-$BASE_CHANNEL.yaml charm/charmcraft.yaml
+  cp charmcraft-$CHARM_TYPE-$BASE_CHANNEL${EXTRA:+}.yaml charm/charmcraft.yaml
```

Comment:
would there be value in inspecting the packed charm to see whether it really is "all"?

Reply:
Yeah probably, at least insofar as "it doesn't take long and it's another place where we're checking this." Good thought! Added.